lib/bibliothecary/parsers/maven.rb

```diff
@@ -425,7 +425,6 @@ def self.property_value(xml, property_name, parent_properties)
         # the xml root is <project> so lookup the non property name in the xml
         # this converts ${project/group.id} -> ${group/id}
         non_prop_name = property_name.gsub(".", "/").gsub("project/", "")
-        return "${#{property_name}}" if !xml.respond_to?("properties") && parent_properties.empty? && xml.locate(non_prop_name).empty?
```

Comment:
to confirm: before, we were returning "${some_property}" if there were no props and no parent props, but now we'll return nil?

if that's correct, the downside seems like it'll be harder to debug uninterpolated props?

Reply:
Right. But extract_property (which is the only thing that calls property_value) has a thing to handle property_value being a nil return (falling back to the value).

This is how the code originally worked but in some refactoring along the way, it got confused